File: Task-03/Javascript/Sub-Task-3.js
function printDiamond(rows) {
    let diamond = '';
    for (let i = 1; i <= rows; i += 2) {
        let spaces = ' '.repeat((rows - i) / 2);
        let stars = '*'.repeat(i);
        diamond += spaces + stars + spaces + '\n';
    }
    for (let i = rows - 2; i > 0; i -= 2) {
        let spaces = ' '.repeat((rows - i) / 2);
        let stars = '*'.repeat(i);
        diamond += spaces + stars + spaces + '\n';
    }
    console.log(diamond);
}

const rows = parseInt(prompt('Enter the number of rows:'));
printDiamond(rows);

File: Task-03/Python/Sub-Task3.py
def pyramid(rows):
    for i in range(0,rows,2):
        print(" " * (rows - i - 1) + "* " * (i + 1))
    for j in range(rows - 2, 0, -2):
        print(" " * (rows - j) + "* " * j)
rows = int(input("Enter the number of rows: "))
pyramid(rows)
